### PR TITLE
feat([sc-9893]): init blog post

### PR DESCRIPTION
**Pull request by plitzenberger**
This PR intends to add the first blog post draft introducing the collaboration and discussing the benefits of synchronizing development by standardization.

### CODE REVIEW

File: docs/pages/blog/embracing-the-future.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+## Why ICP Wallet Standards are Important
+
+### Empowering User Choice in Web3
+
+In the ever-evolving landscape of Web3, one principle stands paramount: user choice. This ethos not only advocates for freedom in wallet selection but also demands flexibility in interaction with various applications. The current state of ICP application development, however, presents a significant hurdle. The need for individual wallet integrations consumes vast amounts of time and resources, diverting attention from enhancing application functionality. Recognizing this, DFINITY and the NFID Wallet team are spearheading an initiative to standardize cryptographic flows, thereby broadening user choices and smoothing the developer experience.
```
Comment (domwoe) [inline docs/pages/blog/embracing-the-future.md:17]: You should mention the Identity & Wallets Standards Working Group and the foundational work by Papers/AirGap.

File: docs/pages/blog/embracing-the-future.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+The `icrc34_get_delegation` method is designed to facilitate the delegation of identity management. This process allows users to securely manage their identities and permissions, providing a flexible and secure framework for identity delegation within the ICP ecosystem.
+
+```typescript
+const response = await connection.getDelegation({...})
+```
+
+## Conclusion
+
+In conclusion, the introduction of ICP wallet standards is a significant stride towards a more unified and user-friendly Web3 environment. These standards not only alleviate the developmental challenges but also enrich the user experience, paving the way for a more interconnected and secure digital future.
```
Comment (domwoe) [inline docs/pages/blog/embracing-the-future.md:128]: I'd add references/links to the Working Group and the standards at the end of the post

File: docs/pages/blog/embracing-the-future.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  version: string // "1"
+  principals: string[] // ["gyu2j-2ni7o-o6yjt-n7lyh-x3sxq-zh7hp-sjvqe-t7oul-4eehb-2gvtt-Jae"]
+}
+
+const response: Response = await connection.getPrincipals()
+```
+
+### ICRC-33 Call Canister
+
+This standard allows applications to request the execution of calls to third-party canisters by the signer. By using the `icrc33_call_canister` method, applications can interact with a broader range of services and functionalities, enhancing the versatility and reach of ICP-based applications.
```
Comment (frederikrothenberger) [inline docs/pages/blog/embracing-the-future.md:103]: I think this is underselling the benefits:
* applications get to make canister calls under a _shared_ identity, some example use-cases would definitely help (in this section or further above)
  * i.e. have all your ICP in one / only a few place(s)
  * control a canister using a singer, and have various other applications help you manage it (i.e. a canister wasm installation helper or something 🤷 ) _without_ them being a controller
  * a voting application that does _not_ need to be a hot-key controller of your neurons
  * ...
* it is an implementation of the transaction approval flow, which gives the user much more control over the actions taken with a given identity

File: docs/pages/blog/embracing-the-future.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+### ICRC-34 Get Delegation
+
+The `icrc34_get_delegation` method is designed to facilitate the delegation of identity management. This process allows users to securely manage their identities and permissions, providing a flexible and secure framework for identity delegation within the ICP ecosystem.
+
+```typescript
+const response = await connection.getDelegation({...})
+```
+
+## Conclusion
```
Comment (frederikrothenberger) [inline docs/pages/blog/embracing-the-future.md:126]: I would still mention a few more things:
* We're in the process of finalizing the standards. Now would be a great time to take a look and provide feedback.
* This is a community wide effort. While NFID spearheads the effort and provides the solid foundation to build this identity kit, everybody is welcome to contribute and help speed things up.
* There are also a lot of surrounding work that could be contributed (e.g. the metadata registry), pushing forward in making sure all the relevant infrastructure supports ICRC-21 (might benefit from library support as well), etc...

File: docs/pages/blog/embracing-the-future.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (plitzenberger) [reply docs/pages/blog/embracing-the-future.md:17]: Oh nice, yea let's link everyone involved here. That's awesome

File: docs/pages/blog/embracing-the-future.md (hunk 2 above)
Comment (plitzenberger) [reply docs/pages/blog/embracing-the-future.md]: There is nothing to try yet. This post is meant to communicate our shared effort and kickstart the collaboration with the wider community. As a series, we'll continuously release more blog posts pointing to the first drafts, examples, and more information. How does this sound?

File: docs/pages/blog/embracing-the-future.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+### Streamlining the Development Process
+
+By introducing standard wallet protocols, developers can bypass the cumbersome process of individual wallet integrations. This standardization not only accelerates the development cycle but also opens the door for a wider range of wallet options for users. The outcome is a more robust and user-friendly ecosystem, where developers can focus on innovation rather than integration.
+
+## How ICP Wallet Standards Will Be Used
+
+### Enhancing User Experience with Seamless Integration
+
+Post-implementation of these standards, the most noticeable change for users will be the increased compatibility between applications and a variety of wallets. This means users can manage assets more efficiently and engage with applications in more dynamic ways. Features like secure data sharing across applications without compromising privacy will become a norm.
```
[the remark this answers is not in the document]
Comment (plitzenberger) [reply docs/pages/blog/embracing-the-future.md]: Yes, I love this! Will try to incorporate this.

File: docs/pages/blog/embracing-the-future.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+This standard outlines a foundational protocol for communication between decentralized apps (dapps) and signers. It specifies a set of messages to facilitate this interaction, ensuring a seamless and secure exchange of information. The ICRC-25 aims to create a uniform language for dapps and signers, enhancing interoperability and simplifying the development process.
+
+#### Establish a Signer Connection:
+
+```typescript
+import * as IdentityKit from '@nfid/identity-kit'
+import { NFID, II } from '@nfid/identity-kit/adapter'
+
+const nfid = NFID.config({
```
[the remark this answers is not in the document]
Comment (plitzenberger) [reply docs/pages/blog/embracing-the-future.md:67]: Yeah, I like that idea. So you mean hinting that you could build your signer even outside of the `@nfid/identity-kit` (e.g., within your codebase), and hook it up like this.

File: docs/pages/blog/embracing-the-future.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+  providerUrl: 'https://nfid.one',
+})
+const ii = II.config({
+  providerUrl: 'https://identity.ic0.app',
+})
+
+IdentityKit.config({
+  derivationOrigin: 'https://<canisterId>.ic0.app',
+  appMeta: {
```
[the remark this answers is not in the document]
Comment (plitzenberger) [reply docs/pages/blog/embracing-the-future.md:76]: Yes, it makes sense. Will remove.

File: docs/pages/blog/embracing-the-future.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (frederikrothenberger) [reply docs/pages/blog/embracing-the-future.md:67]: Yes, exactly. 👍

File: docs/pages/blog/embracing-the-future.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+```typescript
+type Response = {
+  version: string // "1"
+  principals: string[] // ["gyu2j-2ni7o-o6yjt-n7lyh-x3sxq-zh7hp-sjvqe-t7oul-4eehb-2gvtt-Jae"]
+}
+
+const response: Response = await connection.getPrincipals()
+```
+
+### ICRC-33 Call Canister
```
[the remark this answers is not in the document]
Comment (plitzenberger) [reply docs/pages/blog/embracing-the-future.md:128]: yes, working on it.

File: docs/pages/blog/embracing-the-future.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (plitzenberger) [reply docs/pages/blog/embracing-the-future.md:67]: what do you think about it now?

File: docs/pages/blog/embracing-the-future.md (hunk 4 above)
Comment (plitzenberger) [reply docs/pages/blog/embracing-the-future.md:189]: Thanks @frederikrothenberger, these are really good points! I'm working on integrating all of this 🙏

**Review by dmitriiIdentityLabs — APPROVED** [review]

**Pull request merged**